com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -234,6 +234,11 @@ public void NetworkShow(ulong clientId)
                 throw new VisibilityChangeException("The object is already visible");
             }
 
+            if (NetworkManager.NetworkConfig.UseSnapshotSpawn)
```

Comment:
Interesting, when I look at this code I'm expecting to see an 'else' where the old way of spawning happens.  So `SnapshotSpawn` doesn't include a call to `SendSpawnCallForObject`?  Could it?

Reply:
I've put in Slack a link to the lucidchart diagram about this "[Thursday PR]". `NetworkObject.Show` calls directly into Snapshot. It then calls into `SendSpawnCallForObject` in `SpawnManager`. The old way, in `SpawnManager` is then disabled, there. That's where the missing `else` is

That design came around because the other flow, `SpawnInternal`, can be put in Snapshot at once (one command for all connections). If I had put the Snapshot hook in `SendSpawnCallForObject` then `SnapshotSystem` would get the same spawn multiple times. Similarly if I had refactored the old ways to be done for all connections at once, I would have touched code I didn't need to and risk breaking things.